Update waf-cdn-create-portal.md (#53664)
Azure/azure-content: articles/web-application-firewall/cdn/waf-cdn-create-portal.md

> [!div class="checklist"]
> * Create a WAF policy

> > * Associate it with a CDN endpoint. Noe that you can only associate WAF policy with endpoints hosted at Azure CDN from Microsft SKU

Reviewer: ```suggestion
> * Associate it with a CDN endpoint. You can only associate a WAF policy with endpoints hosted at the Azure CDN from Microsft SKU.
```

> > * Associate it with a CDN endpoint. You can associate only a WAF policy that has endpoints that are hosted at the Azure CDN from a  Microsft SKU.

Reviewer: ```suggestion
> * Associate it with a CDN endpoint. You can associate a WAF policy only with endpoints that are hosted on the **Azure CDN Standard from Microsoft** SKU.
```

Reviewer: @ktoliver No, not quite correct. See my updates.
